### ISSUE TITLE
`convert_path` is not part of setuptools API and will be removed

### ISSUE THREAD

**Issue by abravalheri** [Nandaka/PixivUtil2#1334]
Importing `setuptools.convert_path` has been deprecated for almost 2 years and is about to be removed.

https://github.com/Nandaka/PixivUtil2/blob/d0bfaaef0a992a4e8c5ee83a06fdf285bd272808/setup.py#L8

In the past `setuptools` would import `convert_path` from one of its dependencies for internal use, but that is no longer true.

**Comment by biggestsonicfan** [comment]
It looks like it imports `convert_path` from `distutils.util` in case of an import error?

**Comment by abravalheri** [comment]
Yes, but in this case, you also lose `find_packages` and `setup` isn't it?

**Comment by abravalheri** [comment]
BTW,I find that `convert_path` is usually not essential for implementing the common functionality.

This is the description of the function:

>  ```python
>     """Return 'pathname' as a name that will work on the native filesystem,
>     i.e. split it on '/' and put it back together again using the current
>     directory separator.  Needed because filenames in the setup script are
>     always supplied in Unix style, and have to be converted to the local
>     convention before we can actually use them in the filesystem.  Raises
>     ValueError on non-Unix-ish systems if 'pathname' either starts or
>     ends with a slash.
>     """
>  ```

But most of the Python functions nowadays already allow you to use `/` for paths even on windows...

It also seems that in this particular `setup.py`, `conver_path` is being used to convert strings that do not contain `/`... so probably the function is not doing much anyway.
